pkg/oci/containerize_test.go

```diff
@@ -0,0 +1,43 @@
+package oci
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func Test_validateLink(t *testing.T) {
+	root := "testdata/test-links"
+
+	tests := []struct {
+		path  string // path of the file within test project root
+		valid bool   // If it should be considered valid
+		name  string // descriptive name of the test
+	}{
+		{"a.txt", true, "do not evaluate regular files"},
+		{"a.lnk", true, "do not evaluate directories"},
+		{"absoluteLink", false, "disallow absolute-path links"},
```

Comment:
Hmm, it appears this is not working on Windows. I am not sure why. Maybe the link must start with a drive? Or maybe the link on Window cannot be absolute? Worst case: just skip this specific test on Windows.

Reply:
I read through the docs and there is a note in there that even paths beginning with a slash are indeed considered relative in windows.  I'll try adding the drive-letter.